### ISSUE TITLE
Search should include row labels

### ISSUE THREAD

**Issue by hdgarrood** [purescript/pursuit#135]
For example, searching for "console" should give `Control.Monad.Eff.Console.log`.

**Comment by kseo** [comment]
Why is this issue labeled as `after-hoogle-5`? Do we need a specific feature of Hoolge 5?

**Comment by hdgarrood** [comment]
Yes, IIRC Hoogle 5 ought to be a bit more language agnostic. Currently we pretend purescript is Haskell for Hoogle.
